packages/cli/test/test-utils.js

```diff
@@ -71,8 +71,9 @@ exports.givenLBProject = function(rootDir, options) {
   const sandBoxFiles = options.additionalFiles || [];
 
   const content = {};
-  if (!options.excludeKeyword) {
-    content.keywords = ['loopback'];
+  if (!options.excludeDependencies) {
```

Comment:
How about we change the logic to remove the `@loopback/core` from `package.json` when the flag enabled? 

```js
if (options.excludeDependencies) {
  // may need some extra check to make sure the `dependencies` exist
  delete content.dependencies['@loopback/core']
}
```
And maybe rename the flag to be more specific, like `excludeLoopbackCore`. 
Also add a few comments to explain its usage: We infer if a project is loopback by checking its dependencies includes `@loopback/core` or not. This flag is created for testing invalid loopback projects.